ARTstor.js

```diff
@@ -34,98 +37,1043 @@
 	***** END LICENSE BLOCK *****
 */
 
-function getData(doc, checkOnly) {
-	var data = {},
-		widgets = doc.getElementsByClassName('MetaDataWidget'),
-		topWidget = null, topZIndex;
-	
-	// Dtermine the top metadata window
-	for (var i=0; i<widgets.length; i++) {
-		
-		var zindex = doc.defaultView.getComputedStyle(widgets[i], null)
-			.getPropertyValue('z-index');
-		if (!zindex && zindex !== 0) continue;
-		
-		if (!topWidget || topZIndex < zindex) {
-			topWidget = widgets[i];
-			topZIndex = zindex;
-		}
-	}
-	
-	if (!topWidget) return false;
-	
-	var fields = topWidget.getElementsByClassName('field'),
-		found = false;
-	for (var i=0; i<fields.length; i++) {
-		var field = ZU.trimInternal(fields[i].textContent);
-		
-		var value = fields[i].nextElementSibling;
-		if (!value || !value.classList.contains('data')) continue;
-		value = ZU.unescapeHTML(ZU.cleanTags(value.innerHTML));
-		
-		if (!field || !value.trim()) continue;
-		
-		if (checkOnly) return true;
-		
-		found = true;
-		data[field] = value;
-	}
-	
-	return found ? data : false;
-}
-
 function detectWeb(doc, url) {
-	//monitor changes to body's direct children. That's where the metadata popup is added
-	Zotero.monitorDOMChanges(doc.body, {childList: true});
 
-	if (getData(doc, true)) {
-		return "artwork";
-	}
+    if (url.match(/iv2/)) {
+
+        return "artwork";
+    }
+    if (url.match(/imagegroup/) && doc.getElementById("ssContentWrap").style.display == "inline") {
+
+        return false;
+    }
+    if (url.match(/imagegroup/) && doc.getElementById("thumbNavSave1").style.display == "block") {
+        Zotero.debug("Save on");
+        return false;
+    }
+    if (url.match(/(S|s)earch/) || url.match(/categories/) || url.match(/imagegroup/) || url.match(/collections/) || url.match(/collaboratoryfiltering/) || url.match(/cluster/)) {
+        if (doc.getElementsByClassName('MetaDataWidgetRoot').length > 0) {
+            return "artwork";
+
+        } else
+        if (doc.getElementById("floatingPlaceHolder").style.display == "block") {
+            return false;
+
+        }
+        return "multiple";
+    } else {
+        return false;
+    }
+
 }
 
-function associateData(newItem, value, field) {
-	if (!value) return;
-	if (typeof value == 'string') value = ZU.trimInternal(value);
-	if (!value) return;
-	
-	newItem[field] = value;
+function decodeSearchData(str) {
+    var searchParam = str;
+    searchParam = searchParam.replace(/(&gt;)/, ":");
```

Comment:
FYI, this only replaces the first copy of `&gt;`, not all of them. use `g` flag if you want to replace all (maybe you don't)